docs/user-guide/deployment-scenarios.md

```diff
@@ -22,15 +22,103 @@ updates](station-firmware-upgrade.md) for example.
+
+![Single Single](../images/scenarios/1_scale_and_availability.jpg)
+
+### Multi conentrator with single LNS
+
+Multiple concentrators are deployed that reach a single network server. Each sensor can reach one or multiple concentrators. Depending on the goal of the deployment (availability and/or reach).
+
+#### Advantages
+
+1. Scalability: this model scales very well. It's a bit more overhead than the single/single
+model as it will have to deduplicate messages coming in from the additional
+concentrators.
+1. Relatively simple
+1. Low cost
+1. Partial redundancy: if the deployment is designed to ensure that each sensor can reach
+at least 2 concentrators, you get higher availability than with single/single.
+1. Reach: if you do deploy for reach, you can get a larger support a larger geographical
+deployment. To combine reach with availability, the cost will raise.
+
+#### Disadvantages
+
+1. Availability: Depending on the deployment, this solution has at least 1 single point of failure: the network server. The result in the worst case is lost of messages.
+1. Reliability: message delivery might be impacted.
+
+#### Recommended settings
+
+| Target        | Setting               |Value|
+|---------------|:----------------------|:--------|
+| Device Twin   |KeepAliveTimeout       |undefined or 0 |
+| Device Twin   |Deduplication          |None           |
+| Device Twin   |GatewayId              |The Id of the Newtork Server |
+
+![Multi Single](../images/scenarios/2_scale_and_availability.jpg)
+
+### Multi conentrator with Multi LNS
+
+Multiple concentrators are deployed that reach multiple LNS (*note* one concentrator
+can only be connected to a single LNS). Each sensor can reach one or multiple concentrators. Depending on the goal of the deployment (availability and/or reach).
+
+To get high availability concentrator deployment and assignment to LNS should be done
+so that a single sensor is in range of at least 2 concentrators, both connecting to a
+**different** LNS.
+
+#### Advantages
+
+1. Availability: If the deployment is done right, the single point of failures can
+be eliminated.
+1. Reliability: message delivery likelyhood is increased.
+1. Reach: The deployment model can be expanded
+
+#### Disadvantages
+
+1. Cost: all components need to be deployed and managed multiple times.
+1. Complexity
+1. Scalability: The scalability is impacted as the LNS now needs to manage leader election
+for message handling, as well as multiple connetion are opened for the same client on
+multiple edge hubs, resulting in connection ping pong.
+
+#### Recommended settings
+
+| Target        | Setting               |Value|
+|---------------|:----------------------|:--------|
+| Device Twin   |KeepAliveTimeout       | > 0 - depending on the sensor, this can be tuned
+to be the time you expect a new message to come in. Ideally not > 300 |
+| Device Twin   |Deduplication          |Drop           |
```

Comment:
Should we give explanations why Mark/None could give issues?

Reply:
same here as above.